docs/agents/sematext-agent/kubernetes/helm.md

```diff
@@ -0,0 +1,108 @@
+## Introduction
+
+This chart installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
+
+## Prerequisites
+
+- Kubernetes 1.13+
+- You need to create [an Infra App in Sematext Cloud](https://apps.sematext.com/ui/monitoring-create/app/infra) to get your infra token.
+
+## Installation
+
+To install the chart for monitoring and shipping logs run the following command:
+
+```bash
+$ helm repo add sematext https://cdn.sematext.com/helm-charts/
+$ helm install st-agent \
+    --set infraToken=YOUR_INFRA_TOKEN \
+    --set region=US \
+    sematext/sematext-agent
+```
+
+To install the chart for both logs and monitoring run the following command:
+
+```bash
+$ helm repo add sematext https://cdn.sematext.com/helm-charts/
+$ helm install st-agent \
+    --set logsToken=YOUR_LOGS_TOKEN \
+    --set infraToken=YOUR_INFRA_TOKEN \
+    --set region=US \
+    sematext/sematext-agent
+```
+
+To provide your infra token as a kubernetes secret instead, create a secret with `infra-token` as a key, and provide its name to the install command:
+
+```bash
+$ kubectl create secret generic sematext-secret \
+    --from-literal=infra-token=YOUR_INFRA_TOKEN \
+$ helm repo add sematext https://cdn.sematext.com/helm-charts/
+$ helm install st-agent \
+    --set existingSecret.name=sematext-secret \
+    --set existingSecret.hasInfraToken=true \
+    --set region=US \
+    sematext/sematext-agent
+```
+
+
+
+After a few minutes, you should see your services appear in the [Discovery page in Sematext Cloud](https://apps.sematext.com/ui/discovery/services), where you can enable the collection of metrics, events and logs.
+
+**NOTE:** If you want to use Sematext hosted in the EU region set the region parameter to `--set region=EU`. Also, it is worth mentioning that the agent is running as a privileged container.
+
+## Removal
+
+To uninstall the chart use:
+
+
+```bash
+$ helm uninstall st-agent
```

Comment:
st-agent, no sematext-agent?

Reply:
It's the name we give it in the installation example. This can be set to anything the user wants. The actual chart name is the last parameter of the install command: sematext/sematext-agent